runtime-api/src/tests.rs

```diff
@@ -92,7 +92,7 @@ impl pallet_balances::Config for Test {
 	type Balance = Balance;
 	type DustRemoval = ();
 	type RuntimeEvent = RuntimeEvent;
-	type ExistentialDeposit = ConstU64<1>;
+	type ExistentialDeposit = ConstU128<1>;
```

Comment:
What's existential deposit btw?

Reply:
https://wiki.polkadot.network/docs/build-protocol-info#existential-deposit probably describes it best

Reply:
Thanks